### DISCUSSION TITLE
[Script request] Habitica

### DISCUSSION THREAD

**Discussion by stefan131** [community-scripts/ProxmoxVE#878] [Request script] [answered]
### Application Name

Habitica

### Website

https://github.com/HabitRPG/habitica

### Description

Habitica is a productivity app that gamifies task management by turning your to-do list, habits, and goals into a role-playing game. Completing tasks earns rewards like experience points and gear, while neglecting them can cause in-game setbacks. It helps users build good habits, stay organized, and remain motivated through a fun and engaging system.

If this is considered a game-related request then please accept my apology and delete my request. To my understanding this is not a game but a productivity app with several mechanisms for positive feedback that makes it feel like a game.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by kylepyke** [comment]
A quick note that this is a self-hosted version of Habitica:
https://github.com/awinterstein/habitica

**Comment by tremor021** [comment]
Request fulfilled after #2779 merges

**Comment by MickLesk** [comment, marked as answer]
done

**Reply by stefan131** [reply]
Oh my god thank you!! 😍 
Will be testing tonight!

**Reply by aadityajshah** [reply]
Has the script been removed?

**Reply by tremor021** [reply]
yes

**Reply by aadityajshah** [reply]
Any reason?

**Reply by tremor021** [reply]
https://github.com/community-scripts/ProxmoxVE/pull/9489
